saleor/checkout/base_calculations.py

```diff
@@ -0,0 +1,63 @@
+"""Contain functions which are base for calculating checkout properties.
+
+It's recommended to use functions from calculations.py module to take in account plugin
+manager.
+"""
+
+from typing import TYPE_CHECKING
+
+from prices import TaxedMoney
+
+from ..core.taxes import quantize_price, zero_taxed_money
+from ..extensions.manager import get_extensions_manager
+
+if TYPE_CHECKING:
+    from .models import Checkout, CheckoutLine
```

Comment:
It's complete mystery for me when flake8 thinks those imports are used and when not.

Reply:
Yes... Same trouble with pylint in other projects. I think @korycins was putting `# noqa` for flake if those are errors and not install errors from the linter itself.

Maybe we should open and check for an issue to add support of `exclude_lines` setting that `coverage.py` provides. So we could exclude those type checking blocks.

That would be an easier temporary fix as as far as I remember, the issue has been there for a long time and no fix is in sight.

But maybe that's not possible for flake8 to have such filtering.

Reply:
It does not consider them used if you refer to them as a string.